### PR TITLE
Added: Cart management

### PR DESCRIPTION
**Pull request by KaustubhM1997**
Live Preview - https://imaginative-pixie-e64808.netlify.app/

### CODE REVIEW

File: src/pages/Cart/cart.css [hunk 1]
```diff
@@ -1,5 +1,188 @@
 @import url(../../main.css);
 
-.title {
-  margin-top: 8rem;
+:root {
+  --light-grey: #9ca3af;
+  --button-hover: #be185d;
+  --text-green: #16a34a;
+  --box-shadow: 0px 0px 5px 5px var(--light-grey);
```
Comment (bipin7yadav) [inline src/pages/Cart/cart.css:7]: you can use rem instead of pixels to maintain consistency

**Review by bipin7yadav — APPROVED** [review]
good to go

File: src/pages/Cart/cart.js [hunk 2]
```diff
@@ -1,18 +1,83 @@
...
-        <h2 className="h2">Your Cart</h2>
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "flex-start",
+          }}
+        >
+          {/* // style = {{display: "flex", justifyContent: "flex-start", alignItems: "center", */}
+
```
Comment (BelideNikhil) [inline src/pages/Cart/cart.js]: you can remove this if not needed

File: src/pages/Cart/cart.js [hunk 3]
```diff
@@ -1,18 +1,83 @@
...
+            cartlist.length > 0
+              ? cartlist.map((item) => {
+                  //whenever there's an item in the cart (cartlist), just map its details on the cartcard component
+                  return <CartCard key={item._id} productcard={item} />;
+                })
+              : ""
+            //  (
+            //   <h4>Your Cart is empty</h4>
+            // )
+          }
```
Comment (BelideNikhil) [inline src/pages/Cart/cart.js]: here as well, also try to keep the comments short, maybe 1 liners

**Review by BelideNikhil — APPROVED** [review]
The code is nicely written and the UI works as needed
Few suggestions: extract network calls into services and action types into a variable
Also you can do one function call for both increase and decrease.
Remove the console logs and make 1 line comments

**Pull request merged**